scripts/generate-rtk-apis.ts

```diff
@@ -0,0 +1,27 @@
+// Generates Redux Toolkit API slices for certain APIs from the OpenAPI spec
+import type { ConfigFile } from '@rtk-query/codegen-openapi';
+
+const config: ConfigFile = {
+  schemaFile: '../public/openapi3.json',
+  apiFile: '', // leave this empty, and instead populate the outputFiles object below
```

Comment:
i guess we're doing this with an eye on the future where we might output multiple different generated api files? 🤔